os/update-strategies.md

```diff
@@ -89,8 +89,14 @@ coreos:
       command: stop
     - name: locksmithd.service
       command: stop
+    - name: update-engine.service
+      command: mask
+    - name: locksmithd.service
+      command: mask
 ```
 
+Alternatively, to manually mask update-engine run `systemctl mask update-engine.service`. To reverse this remove the stanza from the cloud-config and run `systemctl unmask update-engine.service`
```

Comment:
"remove the stanza from the cloud-config" This is not possible on many platforms. We should stop suggesting that cloud-init be used for configuration management. It's really only good for static configuration.